### ISSUE TITLE
Rack elevations Grouped view for site racks or Rack groups summary

### ISSUE THREAD

**Issue by thefreakquency** [digitalocean/netbox#1949]
### Issue type
[x] Feature request 
[ ] Bug report   
[ ] Documentation   

### Environment
* Python version:  3.5.4
* NetBox version:  2.3.1

### Description
It would be beneficial to be able to see rack elevations for a given rack group on the rack group page, under or above the rack listing. 

**Use case:**
Our sites contain one or a few groups, but group names are the same for multiple sites (i.e.: Shelter, roof-top, Cage1... ). Users are usually accessing the site page, then the site group from there. They have to go then in each racks, but are not provided an overview for the group from there. 

Elevations for rack groups are available by going to Racks/Elevation, then filter by rack group. When they go in the rack Elevation view, in our case, they can only filter by site name since rack groups names are repeated multiple time in the filter options... but then they are presented with all racks at a given site. 

**Note:**
This FR was partially addressed (Thanks @jeremystretch !) in #1801 but only for the /dcim/rack-elevations/ page. It does not provide grouped elevation views to the end user in other way, especially if their workflow is starting at the site level as mentioned in the use case. By example: from /dcim/sites/ pick a site name. On the site page, you can click on Racks (in Stats), or on a site group name in the Rack Groups section, but it does not provide the grouped elevation view. You would have to go to Racks, Elevations and apply filters again to obtain the rack elevations for a given site group or site... 

In short, I see 2 intuitive ways this could be approached. Either by:
- Embedding rack elevations grouped view on the following pages:
  -  /dcim/racks/?group_id=xx  
  -  /dcim/racks/?site=SITENAME 

- Add additional links (or buttons ) on the site page ( /dcim/sites/SITENAME/ ) to redirect the elevation pages with the relevant filters already applied (ie: /dcim/rack-elevations/?q=&site=SITENAME&group_id=xx  I.e.: The links could be added :
  - next to Site Group Name 
  - next to Rack count in Stats
  - in a new sub-section of the site page

**Comment by jeremystretch** [comment]
> It would be beneficial to be able to see rack elevations for a given rack group on the rack group page, under or above the rack listing.

There is no "rack group page;" there is only the racks list view which can be filtered by site and group.

We already have a view which show racks elevations. We can add a link to the elevations view for each group to the rack groups table (`/dcim/rack-groups/`), but we're not going to embed the elevations in a different view as that would be redundant.

**Comment by thefreakquency** [comment]
Sorry if it wasn't clear enough. Probably got lost in translation.

By "rack group page" I meant `/dcim/racks/?group_id=xx` which is the result of clicking of a rack group name from the site page. This is page is the same as if you'd go in the rack groups table (`/dcim/rack-groups/`) and click on a rack group name. 

Since rack groups are limited to a given site, is there a way to have either a link to elevations or embed those elevations on the site page (IE: on a different tab maybe) ? This would complement a link to elevations on the `/dcim/racks/?group_id=xx` pages and cover most use cases. 

My goal here is to simplify the end-users workflows. Most of them will drill down from regions or sites and will want to visualise site content (ie: devices, racks, racks elevations, etc...) instead of going to rack elevations (`/dcim/rack-elevations/`) and filter to see the elevations, then navigate and go back 

I hope this clarification make sense to you.

**Comment by jeremystretch** [comment]
Yeah, adding a link is sufficient. No need to replicate the entire view.
